deployment/I_exrm_releases.md

```diff
@@ -165,88 +165,142 @@ Everything related to our releases is in the `rel/hello_phoenix` directory. Let'
 
 ```console
 $ ls -la rel/hello_phoenix/
-total 27216
-drwxr-xr-x   7 lance  staff       238 May 13 18:47 .
-drwxr-xr-x   3 lance  staff       102 May 13 18:47 ..
-drwxr-xr-x   6 lance  staff       204 May 13 18:47 bin
-drwxr-xr-x   8 lance  staff       272 May 13 18:47 erts-6.4
--rw-r--r--   1 lance  staff  13933031 May 13 18:47 hello_phoenix-0.0.1.tar.gz
-drwxr-xr-x  26 lance  staff       884 May 13 18:47 lib
-drwxr-xr-x   5 lance  staff       170 May 13 18:47 releases
+drwxrwxr-x  6 sashaafm sashaafm 4096 Mar  1 10:29 .
```

Comment:
might want to make this a generic user, unless you want everyone to find the docs when they search your name :P